client/src/components/employer-table-component/index.jsx

```diff
@@ -0,0 +1,318 @@
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <SearchIcon />
+                      </InputAdornment>
+                    ),
+                  }}
+                />
+              </Typography>
+              <Typography
+                sx={{ fontSize: 14, mb: 2 }}
+                color="text.secondary"
+                align="left"
+                gutterBottom
+              >
+                Phone Number
+              </Typography>
+              <Typography sx={{ mb: 2 }} color="text.secondary">
+                <TextField
+                  type="text"
+                  size="small"
+                  style={{
+                    borderWidth: "10px",
+                  }}
+                  InputProps={{
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <SearchIcon />
+                      </InputAdornment>
+                    ),
+                  }}
+                />
+              </Typography>
+              <Typography
+                sx={{ fontSize: 14, mb: 2 }}
+                color="text.secondary"
+                align="left"
+                gutterBottom
+              >
+                Date Added
+              </Typography>
+              <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <DatePicker
+                  label="From"
+                  renderInput={(params) => (
+                    // eslint-disable-next-line
+                    <TextField {...params} error={false} helperText="" />
+                  )}
+                  sx={{ mb: 2 }}
+                />
+              </LocalizationProvider>
+
+              <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <DatePicker
+                  label="To"
+                  renderInput={(params) => (
+                    // eslint-disable-next-line
+                    <TextField {...params} error={false} helperText="" />
+                  )}
+                  sx={{ mb: 2 }}
+                />
+              </LocalizationProvider>
+              <Typography
+                sx={{ fontSize: 14, mb: 2 }}
+                color="text.secondary"
+                align="left"
+                gutterBottom
+              >
+                Postal Code
+              </Typography>
+              <Typography color="text.secondary">
+                <TextField
+                  type="text"
+                  size="small"
+                  style={{
+                    borderWidth: "10px",
+                  }}
+                  InputProps={{
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <SearchIcon />
+                      </InputAdornment>
+                    ),
+                  }}
+                />
+              </Typography>
+            </CardContent>
+          </Card>
+          <TableContainer
```

Comment:
I believe because we are planning on doing server side pagination, we need to use mui datagrid instead of table here is a link to the documentation https://mui.com/x/react-data-grid/, but you can also follow how i set it up in my pr similar to my pr here https://github.com/uoftblueprint/tces/pull/41